src/methods/using_methods/hard/q4/HowOld.java

```diff
@@ -0,0 +1,38 @@
+package methods.using_methods.hard.q4;
+
+/*
+Write a Java Program called HowOld that asks users for the current year and for the year that they were born in.
```

Comment:
Change question text to:
`
Write a Java Program called HowOld that asks users for the current year and the year they were born. Then, write a method called ageCalc that accepts both values as input parameters and calculates and returns the user's age. This method should check that both years are positive and that the current year is greater than or equal to the year the user was born. If not, return -1.
`